### PR TITLE
chore: change lerna with yarn workspace

### PR DESCRIPTION
**Pull request by BeroBurny**

### CODE REVIEW

**Comment by mpetrunic** [comment]
You need to update CI :)

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -26,13 +26,12 @@ jobs:
...
       - run: cargo clippy --release --all-features --target wasm32-unknown-unknown -- --deny warnings
       - run: wasm-pack test --node --release --all-features
-      - run: lerna bootstrap
-      - run: lerna run build --stream
-      - run: lerna run test --stream
-      - run: lerna run ci --stream
-      - run: lerna run benchmark --stream
-      - run: wasm-pack build -t web -d pkg-web -s hanabi1224
-      - run: wasm-pack build -t nodejs -d pkg-node -s hanabi1224
+      - run: yarn install
```
Comment (mpetrunic) [inline .github/workflows/ci.yml:29]:
```suggestion
      - run: yarn install --immutable
```
otherwise lockfile makes no sense

**Review by mpetrunic — APPROVED** [review]
otherwise LGTM

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -26,13 +26,12 @@ jobs:
...
+      - run: yarn workspaces foreach -vpt run build
+      - run: yarn workspaces foreach -vpt run test
```
[the remark this answers is not in the document]
Comment (BeroBurny) [reply .github/workflows/ci.yml:31]: Rome is a formatter, linter, bundler, and [more](https://rome.tools/#development-status) for JavaScript, TypeScript, JSON, HTML, Markdown, and CSS.

and `lerna run ci` runs `rome ci *.[jt]s` inside `integration_tests` i dont see reason for using it in ci

**Pull request merged**